### DISCUSSION TITLE
Custom name for generated pdf?

### DISCUSSION THREAD

**Discussion by anna-arlig** [pdfme/pdfme#147] [Q&A]
When I generate a pdf it gets a name that is just a bunch of numbers and letter like so: "b115ac76-92e0-408f-94c8-642825375b79". Can I configure my own naming standard? Something more like: "companyName + projectName"

**Comment by gxanshu** [comment]
well ofcourse you can 
The base64 representation of the file only contains the content of the file, but not the file name. But you can manually assign a file name this way: 
```javascript
function download() {
    const source = 'data:application/pdf;base64,'+ generatedPdfBase64;
    // generatedPdfBase64 = 'JVBERi0xL....(more)'
    const downloadLink = document.createElement("a");
    const fileName = 'file.pdf'; // here you can customize name

    downloadLink.href = source;
    downloadLink.download = fileName;
    downloadLink.click();
}
```
